frontend/.env

```diff
@@ -0,0 +1,2 @@
+REACT_APP_API=http://localhost:8000/api
+REACT_APP_THUMBNAIL=http://127.0.0.1:8000/storage/
```

Comment:
I see na apil c `.env` sa imo commit. Suppose to be dili siya makita / maapil as commit changes.

try this to update apply the `.gitignore` changes
https://stackoverflow.com/a/25436481

Reply:
Done updating the commit Sir.